code/modules/clothing/suits/jobs.dm

```diff
@@ -155,7 +155,25 @@
 	name = "surgical apron"
 	desc = "A sterile blue surgical apron."
 	icon_state = "surgical"
-	allowed = list(/obj/item/scalpel, /obj/item/surgical_drapes, /obj/item/cautery, /obj/item/hemostat, /obj/item/retractor)
+	allowed = list(
+		/obj/item/hemostat,
+		/obj/item/cautery,
+		/obj/item/scalpel,
+		/obj/item/surgical_drapes,
+		/obj/item/retractor,
+		/obj/item/bonesetter,
+		/obj/item/scalpel/advanced,
+		/obj/item/retractor/advanced,
```

Comment:
Aren't these redundant? I assume that the allowed suit slot checks for subtypes.

Reply:
apologies for the late reply i was doing... stuff.

tested locally, those two lines were indeed redundant. Removed.